x-pack/plugins/observability/public/embeddable/slo/alerts/components/slo_alerts_table.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { AlertConsumers } from '@kbn/rule-data-utils';
+import type { TimeRange } from '@kbn/es-query';
+import { SloEmbeddableDeps } from '../slo_alerts_embeddable';
+import type { SloItem } from '../types';
+
+type SloIdAndInstanceId = [string, string];
+const ALERTS_PER_PAGE = 10;
+const ALERTS_TABLE_ID = 'xpack.observability.sloAlertsEmbeddable.alert.table';
+
+interface Props {
+  deps: SloEmbeddableDeps;
+  slos: SloItem[];
+  timeRange: TimeRange;
+  onLoaded?: () => void;
+}
+
+export function SloAlertsTable({ slos, deps, timeRange, onLoaded }: Props) {
+  const {
+    triggersActionsUi: { alertsTableConfigurationRegistry, getAlertsStateTable: AlertsStateTable },
+  } = deps;
+  const slosWithoutName = slos.map((slo) => ({
+    id: slo.id,
+    instanceId: slo.instanceId,
+  }));
+  const sloIdsAndInstanceIds = slosWithoutName.map(Object.values) as SloIdAndInstanceId[];
+
+  return (
+    <AlertsStateTable
+      query={{
+        bool: {
+          filter: [
+            {
+              range: {
+                '@timestamp': {
+                  gte: timeRange.from,
+                },
+              },
+            },
+            {
+              term: {
+                'kibana.alert.rule.rule_type_id': 'slo.rules.burnRate',
+              },
+            },
+            {
+              bool: {
+                should: sloIdsAndInstanceIds.map(([sloId, instanceId]) => ({
+                  bool: {
+                    filter: [
+                      { term: { 'slo.id': sloId } },
+                      { term: { 'slo.instanceId': instanceId } },
+                    ],
+                  },
+                })),
+              },
+            },
+          ],
+        },
+      }}
+      alertsTableConfigurationRegistry={alertsTableConfigurationRegistry}
+      configurationId={AlertConsumers.SLO}
+      featureIds={[AlertConsumers.SLO, AlertConsumers.OBSERVABILITY]}
```

Comment:
@XavierM What are the correct featureIds I need to pass in here? Are both `slo` and `observability` needed?

Reply:
Relates to https://github.com/elastic/kibana/pull/168636#issuecomment-1792154543

Reply:
This is LGTM to me